src/cli/index.ts

```diff
@@ -32,12 +32,19 @@ const roundsCommand = (_args: { [key: string]: string }) =>
     .then((rounds) => rounds.forEach((r) => console.log(r)))
     .catch(logError);
 
-const projectCommand = (args: { [key: string]: string }) =>
+const projectCommand = (_args: { [key: string]: string }) =>
```

Comment:
why are we adding the `_`? 
I usually add it to remove lint warning if the variable is not used. 
Since commands have all same signature we always have the parameter but some of them like `projectsCommand` are not using args. 
When we use it I would recommend to have `args` instead of `_args`